stories/src/data/tableGraph.ts

```diff
@@ -4,8 +4,8 @@ export const tableCSV = `#group,false,false,true,true,false,false,true,true,true
 ,result,table,_start,_stop,_time,_value,_field,_measurement,cpu,host
 ,,0,2020-07-01T21:45:43.0968Z,2020-07-01T21:50:43.0968Z,2020-07-01T21:45:50Z,2.2,usage_system,cpu,cpu1,MBP15-TLUONG.local
 ,,0,2020-07-01T21:45:43.0968Z,2020-07-01T21:50:43.0968Z,2020-07-01T21:46:00Z,1,usage_system,cpu,cpu1,MBP15-TLUONG.local
-,,0,2020-07-01T21:45:43.0968Z,2020-07-01T21:50:43.0968Z,2020-07-01T21:46:10Z,1.2,usage_system,cpu,cpu1,MBP15-TLUONG.local
-,,0,2020-07-01T21:45:43.0968Z,2020-07-01T21:50:43.0968Z,2020-07-01T21:46:20Z,0.8,usage_system,cpu,cpu1,MBP15-TLUONG.local
+,,0,2020-07-01T21:45:43.0968Z,2020-07-01T21:50:43.0968Z,2020-07-01T21:46:10Z,12123,usage_system,cpu,cpu1,MBP15-TLUONG.local
+,,0,2020-07-01T21:45:43.0968Z,2020-07-01T21:50:43.0968Z,2020-07-01T21:46:20Z,2345678,usage_system,cpu,cpu1,MBP15-TLUONG.local
```

Comment:
Thanks for adding the two lines that will show the commas.

I don't think we should replace any of the existing data, though. The existing data will help with identifying any issues with formatting decimal places (regression). The table will work just fine with more lines of data.